misk/src/test/kotlin/misk/metrics/backends/stackdriver/StackDriverReporterTest.kt

```diff
@@ -0,0 +1,290 @@
+package misk.metrics.backends.stackdriver
+
+import com.codahale.metrics.Gauge
+import com.google.api.client.util.DateTime
+import com.google.api.services.monitoring.v3.model.TimeSeries
+import com.google.common.truth.Truth.assertThat
+import com.google.inject.AbstractModule
+import com.google.inject.Provides
+import misk.config.AppName
+import misk.environment.InstanceMetadata
+import misk.metrics.Metrics
+import misk.metrics.MetricsModule
+import misk.testing.MiskTestRule
+import misk.time.FakeClock
+import misk.time.FakeClockModule
+import org.junit.Rule
+import org.junit.Test
+import java.math.BigDecimal
+import java.math.BigInteger
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+import javax.inject.Singleton
+
+internal class StackDriverReporterTest {
+    companion object {
+        @JvmStatic
+        val APP_NAME = "my_app"
```

Comment:
If these are just being used to test the `StackDriverReporter` class, they shouldn't need `@JvmStatic`.